src/commands/remove.js

```diff
@@ -53,15 +53,20 @@ module.exports = async function (context) {
   // take the last parameter (because of https://github.com/infinitered/gluegun/issues/123)
   const moduleParam = parameters.array.pop()
 
+  const isScoped = /^@/.test(moduleParam)
```

Comment:
```suggestion
  const isScoped = moduleParam.startsWith('@')
```

I'd prefer the more understandable `startsWith` function here.